packages/yourloops/test/integration/use-cases/patient-data-visualisation.ts

```diff
@@ -61,7 +61,10 @@ export const testDashboardDataVisualisationForPatient = async (patientDashboardL
   await checkDeviceUsageWidget()
 }
 
-export const testDashboardDataVisualisationWithTwoWeeksOldData = async () => {
+export const testDashboardDataVisualisationWithOldData = async () => {
+  await checkPatientStatisticsWithSixteenDaysOldData()
+}
+export const testDashboardDataVisualisationTwoWeeksOldData = async () => {
```

Comment:
<!-- Suggestions propose improvements to the current subject. It's important to be explicit and clear on what is being suggested and why it is an improvement. Consider using patches to further communicate your intent.  -->
**🎯 suggestion (non-blocking):** ‏Missing line return between 2 functions